dateparser/date.py

```diff
@@ -120,11 +122,18 @@ def get_date_from_timestamp(date_string, settings, negative=False):
          match = RE_SEARCH_TIMESTAMP.search(date_string)
 
     if match:
+        if settings is not None and settings.TIMEZONE is not None and 'local' not in settings.TIMEZONE.lower():
```

Comment:
```suggestion
        if (
            settings is not None 
            and settings.TIMEZONE is not None 
            and 'local' not in settings.TIMEZONE.lower()
        ):
```

This way it seems more readable. I had trouble initially understanding the whole statement. So I believe this can be restructured a bit.

Reply:
how about flipping the boolean logic, like I mentioned in this comment: https://github.com/scrapinghub/dateparser/pull/954#discussion_r958668351 ? I feel like that might be clearer. I could change the formatting and logic.

Reply:
The latest push has the change in formatting you suggested, clarifying comments, and flips the boolean logic so that I think it's clearer.